variables.tf

```diff
@@ -19,6 +19,12 @@ variable "delimiter" {
   description = "Delimiter to be used between `namespace`, `stage`, `name` and `attributes`"
 }
 
+variable "use_fullname" {
+  type        = "string"
+  default     = "true"
+  description = "Set 'true' to use `cp-prod-bastion_image` for ecr repository name, else `bastion_image`"
```

Comment:
update to something like this:

Set to '"true"' to use the `namespace-stage-name` format for the ECR repository name, otherwise just the `name` part will be used